src/main/scala/controllers/ActionHandler.scala

```diff
@@ -102,27 +102,22 @@ class ActionHandler @Inject()(applications: ApplicationOps, applicationForms: Ap
   def redirectToPreview(id: ApplicationId, sectionNumber: Int) =
     Redirect(routes.ApplicationPreviewController.previewSection(id, sectionNumber))
 
-  def renderSectionForm(id: ApplicationId,
+  def renderSectionForm(app: ApplicationDetail,
```

Comment:
Ahh, that's opening up a whole unsettled debate within the Scala world - the use of protected/private scope modifiers.